recipes/limereport/all/test_package/test_package.cpp

```diff
@@ -1,24 +1,15 @@
-#include <QCoreApplication>
-#include <QGuiApplication>
+#include <cstdlib>
 #include <iostream>
 #include <limereport/LimeReport>
 #include <limereport/config.h>
 
 int main(int argc, char* argv[])
 {
-    QGuiApplication a(argc, argv);
```

Comment:
Are you sure it needed to be removed? Because LimeReport engine cannot be created without QGuiApplication

Reply:
@Jihadist We dont need really create an application, only check the package consistence. If we can build a very small application that proves we are able to build and run, that's enough.

Reply:
Ok. I did this because old recipes contain runnable test_packages.